### DISCUSSION TITLE
Build Fail due to Two Different Operation Options Specified

### DISCUSSION THREAD

**Discussion by cdds-at-uiuc** [E3SM-Project/E3SM#5929] [E3SM model help]
Hello! I currently run E3SM that is ported on a local machine, but came across with this error as I build after setting up the case in the  mct build log: 

`
ERROR: /data/keeling/a/username/miniconda3/bin/x86_64-conda-linux-gnu-ar: two different operation options specified
gmake[1]: *** [libmpeu.a] Error 1
gmake: *** [subdirs] Error 2
`

Unable to resolve the inconsistent package issue a couple days ago, I removed miniconda3 and reinstalled it along with the commonly-used packages on the conda-forge channel like jupyter and nco. However, cases that I created before and after the reinstallation can no longer build due to the above error. I found similar errors posted on the CESM forum, but their solution seems to be related to machine configuration. Since E3SM was running fine before the reinstallation, I am guessing the problem is caused by missing packages in conda rather than an error caused by setups in the config_machine scripts. Would anyone mind advising on how to fix this error? 

Any advice and comments about the error would be much appreciated! Please let me know if more details are needed.

**Comment by rljacob** [comment]
The Case Control System just needs base python.  Try building outside of any conda environment.
